### PR TITLE
[13.0][MIG] document_url : Migration to 13.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
This is the migration of document_url to v 13.0

@Tecnativa TT19733

### CODE REVIEW

**Comment by manuelcalerosolis** [comment]
Ready to be reviewed @pedrobaeza @ernestotejeda @chienandalu 
Please review

File: document_url/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
+#                       Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# Copyright 2020 Tecnativa - Manuel Calero
+{
+    "name": "URL attachment",
+    "version": "13.0.1.0.0",
+    "category": "Tools",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
```
Comment (ernestotejeda) [inline document_url/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

**Review by ernestotejeda — COMMENTED** [review]
Generate readme

**Review by ernestotejeda — COMMENTED** [review]
Functional review: OK
Looks very nice, jeje

File: document_url/wizard/document_url.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
+#                       Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+from urllib import parse
+
+from odoo import fields, models
+
+
+class AddUrlWizard(models.Model):
```
Comment (ernestotejeda) [inline document_url/wizard/document_url.py]:
```suggestion
class AddUrlWizard(models.TransientModel):
```

**Review by ernestotejeda — CHANGES_REQUESTED** [review]

File: document_url/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
+#                       Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```
Comment (chienandalu) [inline document_url/__manifest__.py]: Please, update Pedro's author lines
```suggestion
# Copyright 2014-20 Tecnativa - Pedro M. Baeza
```

**Review by chienandalu — CHANGES_REQUESTED** [review]
Thanks, @manuelcalerosolis :) Did you attend @ernestotejeda remarks

**Review by chienandalu — CHANGES_REQUESTED** [review]
Functional test ok :) Attend comments to unblock the PR :)

File: document_url/view/document_url_view.xml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    <template id="assets_backend" name="google_drive assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <script type="text/javascript" src="/document_url/static/src/js/url.js"></script>
+        </xpath>
+    </template>
+
+    <record model='ir.actions.act_window' id='action_ir_attachment_add_url'>
+        <field name='name'>Add URL</field>
+        <field name='res_model'>ir.attachment.add_url</field>
+        <field name='type'>ir.actions.act_window</field>
```
Comment (Tardo) [inline document_url/view/document_url_view.xml:15]: :fire:
It's the default value

**Review by Tardo — CHANGES_REQUESTED** [review]
Looks good! only one non-blocking comment...

File: document_url/security/ir.model.access.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_ir_attachment_add_url,access_ir_attachment_add_url,model_ir_attachment_add_url,,1,1,1,1
```
Comment (chienandalu) [inline document_url/security/ir.model.access.csv]:
```log
2020-04-22 14:55:50,976 158 WARNING openerp_test odoo.modules.loading: The transient model ir.attachment.add_url (Wizard to add URL attachment) should not have explicit access rules!
```

**Comment by flachica** [comment]
I need this module. I will open new PR.

**Comment by victoralmau** [comment]
Please, @joao-p-marques, @chienandalu , @Tardo, @ernestotejeda review it again

**Review by ernestotejeda — COMMENTED** [review]
Code review: :+1: 
Regenerate the readme

**Review by victoralmau — APPROVED** [review]

**Review by ernestotejeda — APPROVED** [review]
Functional review: :+1:

**Review by joao-p-marques — APPROVED** [review]
LGTM :+1: 

Code and functional review

**Pull request merged**